packages/react-component-library/src/components/Timeline/Months.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { format, getDaysInMonth } from 'date-fns'
+
+import { getKey } from './helpers'
+import { TimelineContext } from './context'
+import { DAY_WIDTH } from './constants'
+
+export type MonthsProps = ComponentWithClass
+
+export const Months: React.FC<MonthsProps> = () => {
+  return (
+    <TimelineContext.Consumer>
+      {({ state }) => (
+        <div className="timeline__months">
+          {state.months &&
+            state.months.map(({ startDate }, index) => {
+              const days = getDaysInMonth(startDate)
+
+              return (
+                <div
+                  className="timeline__month"
+                  key={getKey('timeline-month', index)}
+                  style={{
+                    width: `${days * DAY_WIDTH}px`,
+                  }}
+                >
+                  <span className="timeline__month-title">
+                    {format(startDate, 'MMMM yyyy')}
```

Comment:
I wonder if we should have some global constants with this date format in? ie `MMMM yyyy`